### DISCUSSION TITLE
We need a detailed tutorial about training

### DISCUSSION THREAD

**Discussion by yuananf** [coqui-ai/TTS#663] [General]
We need something like this(better with an updated colab notebook):
https://awesomeface349.gitbook.io/uberduck-written-tutorial/training-time/simplified-training-page

With this tutorial I'm able to train a new model in a couple of hours.

And if possible, a discord channel will be easy to communicate.

**Comment by erogol** [comment]
Have you checked https://tts.readthedocs.io/en/latest/ ?

**Reply by yuananf** [reply]
I'm totally new to TTS, so here are some problems.
1. Do I need to train Text2Spec and Vocoder from scratch if I want to clone my own voice? Or just Text2Spec model is enough?  
2. Where can I find correct notebooks to train a model? Because I tried some notebooks but it seems the code structure has changed.
3. The config.json is very complicated, and I really don't know which parameters should I change when training a new model.

**Reply by erogol** [reply]
> I'm totally new to TTS, so here are some problems.
> 
> 1. Do I need to train Text2Spec and Vocoder from scratch if I want to clone my own voice? Or just Text2Spec model is enough?

Currently, we assume you know that much before you use :frog: TTS. Hopefully, we add docs for complete beginners not only for 🐸 TTS but TTS in general.

The short answer is, yes you need these two models either from scratch or fine-tuning a model depending on your dataset and the language.

I suggest you go over all the things under Using TTS in the docs. 

> 2. Where can I find correct notebooks to train a model? Because I tried some notebooks but it seems the code structure has changed.
> 3. The config.json is very complicated, and I really don't know which parameters should I change when training a new model.

Both these questions are partially answered here https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html

**Reply by yuananf** [reply]
Thanks for your explaining, I will look at the doc for more details.
